src/components/SRNEntry.vue

```diff
@@ -250,50 +293,45 @@ export default {
             authType
           );
     },
-    //method that authentiates the SRN
+    /* this method is called whenever + button is clicked. It authenticates the most recent typed ID. 
+    If the ID is invalid, then the login mesasge displays an error and the input field is cleared for the user to correct their entry.
+    @params {String} - userID - most recent ID  */
     async authenticateSRN(userID) {
       this.isLoading = true;
-      //invokes the validation function
       let userValidationResponse = await validateSRN(
         userID,
         this.validateCount,
         this.isSingleEntryOnly,
         this.redirectID,
-        this.isUserValid,
+        this.isCurrentUserValid,
         this.purpose,
         this.purposeParams,
         this.redirectTo
       );
-      this.isUserValid = userValidationResponse.isUserValid;
+      this.isCurrentUserValid = userValidationResponse.isCurrentUserValid;
       this.validateCount = userValidationResponse.validateCount;
       this.invalidLoginMessage = userValidationResponse.invalidLoginMessage;
       this.isLoading = false;
-      //clear the input field if entry is incorrect
+
       if(this.invalidLoginMessage != ""){
         this.resetEntry(this.userIdListLength - 1)
       }
-      
-
-      
     },
-    //method called after clicking the submit button
+    /* this method is called after the user clicks the submit button. Since all the ID's but the last are authenticated, this method calls authenticateSRN(). 
+    If the user is invalid and is being authenticated the first time, handleIncorrectEntry() is called. 
+    If the user is valid or is being authenticated the second time, the user is reidrected to their destination and an SQS message is sent. */
```

Comment:
I don't see why you need to add the implementation details in the docstring - about which methods to call in which situations. The docstring is meant to give a high-level overview of what the function does!

+ please follow the docstring format suggested above.

Reply:
Agreed. We should just write what the function does and that's it